frontend/src/main/components/CourseSearch/SubjectSelector.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { Form } from 'react-bootstrap';
+import { compareValues } from "main/utils/sortHelper"
+
+const SubjectSelector = ({ subjects, subject, setSubject, controlId, onChange = null, label = "Subject" }) => {
```

Comment:
the subject prop is unused, should probably be removed

Reply:
Agreed.